src/bot.py

```diff
@@ -29,7 +29,7 @@
         | interactions.Intents.GUILD_MESSAGE_CONTENT
         | interactions.Intents.GUILDS
     ),
-    disable_sync=True
+    # disable_sync=True
```

Comment:
hm

Reply:
There is a reason though. Default is ``/mod channel purge channel: length:``, now, ``/mod channel purge amount: channel(optional):``, so it should be uncommented so the command can sync and avoid this.
![image](https://user-images.githubusercontent.com/60958064/178116778-1d38b36b-f770-41de-97cf-4e63ccb53f4c.png)

Reply:
ah i see ok